code/datums/components/atmospheric_scanner.dm

```diff
@@ -0,0 +1,58 @@
+///Atoms with this component can call it in order to get a scan of surrounding atmospherics sent to a specific mob.
```

Comment:
Call it?

Reply:
idk how to word thing i have brain damag

though i guess "signal it" would be a lot better